eel/tests/receiving_payments_test.rs

```diff
@@ -164,14 +167,27 @@ mod receiving_payments_test {
 
             // Add 2M sats of inbound capacity - More than the max allowed receive amount
             let node_id = node.get_node_info().node_pubkey.to_hex();
+            assert_eq!(nigiri::get_number_of_txs_in_mempool(), Ok::<u64, String>(2)); // 2 times jit channel open flow
```

Comment:
Probably this is obvious but I was getting confused why the already open channel would still have TXs in the mempool :) 
```suggestion
            assert_eq!(nigiri::get_number_of_txs_in_mempool(), Ok::<u64, String>(2)); // 2 times jit channel open flow (zero-conf)
```

Reply:
Yes it was obvious to me. Thanks for pointing out. Elaborating makes sense indeed